src/components/__tests__/rating.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { mount } from "enzyme";
+
+import Rating from '../Rating';
+
+describe('Rating', () => {
+    const onStarClick = jest.fn();
+    const ratingProps = {
+        rate_score: 4,
+        onStarClick: onStarClick,
+        averageRating: 4
+    };
+    const rate = mount(<Rating {...ratingProps} />);
+    it("renders rate component correctly", () => {
```

Comment:
I think the message always start with what the test should do. For example, in this case, the 'it' method message should be it ('should renders rate component correctly')